src/app/components/Home/Office/Canvas/PopUpInfoPlace/components/index.ts

```diff
@@ -0,0 +1,21 @@
+import Button from "@material-ui/core/Button";
+import Dialog from "@material-ui/core/Dialog";
+import MuiDialogTitle from "@material-ui/core/DialogTitle";
+import MuiDialogContent from "@material-ui/core/DialogContent";
+import MuiDialogActions from "@material-ui/core/DialogActions";
+import IconButton from "@material-ui/core/IconButton";
+import CloseIcon from "@material-ui/icons/Close";
+import Typography from "@material-ui/core/Typography";
+import TextField from "@material-ui/core/TextField";
+
+export {
```

Comment:
try shorter `export { default as Button } from "@material-ui/core/Button"` in such aggregate files.